### ISSUE TITLE
[SDLAlert init] not initialize class propertis with default values

### ISSUE THREAD

**Issue by pierzyns** [smartdevicelink/sdl_ios#901]
### Bug Report
[SDLAlert init] doesn't initialize class properties with default values. 

##### Reproduction Steps
1. Call [[SDLAlert alloc] init]
2. Check the values of the class properties
3. The class properties are not initialized to default values described in header file SDLAlert.h

##### Expected Behavior
The class properties should be initialized with default values.

##### Observed Behavior
The class properties are not initialized to default values described in header file SDLAlert.h
If it not possible the NS_DESIGNATED_INITIALIZER should be specified.

##### OS & Version Information
* iOS Version: 11.1.1
* SDL iOS Version: v5.1.1
* Testing Against: Ford Sync3 SDL emulator

##### Test Case, Sample Code, and / or Example App
```
- (instancetype)init {
    if (self = [super initWithName:SDLNameAlert]) {
    }
    return self;
}
```
vs
```
- (instancetype)initWithTTSChunks:(nullable NSArray<SDLTTSChunk *> *)ttsChunks alertText1:(nullable NSString *)alertText1 alertText2:(nullable NSString *)alertText2 alertText3:(nullable NSString *)alertText3 playTone:(BOOL)playTone duration:(UInt16)duration softButtons:(nullable NSArray<SDLSoftButton *> *)softButtons {
    self = [self init];
    if (!self) {
        return nil;
    }

    self.ttsChunks = [ttsChunks mutableCopy];
    self.alertText1 = alertText1;
    self.alertText2 = alertText2;
    self.alertText3 = alertText3;
    self.playTone = @(playTone);
    self.duration = @(duration);
    self.softButtons = [softButtons mutableCopy];

    return self;
}
```

**Comment by pierzyns** [comment]
@joeljfischer  So as I understand correctly there is no difference when I call [[SDLAlert alloc] init] or [[SDLALert alloc] initWithAlertText1:nil alertText2:nil alertText3:nil] ? What about properties like duration or playTon which according to the spec are initialized with some default values.

**Comment by joeljfischer** [comment]
That's correct. The default values mean that if you don't set a value, the system will assume those are the values.
